launch_ros/launch_ros/actions/composable_node_container.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the ComposableNodeContainer action."""
+
+from typing import List
+from typing import Optional
+
+from launch.action import Action
+from launch.launch_context import LaunchContext
+
+from .node import Node
+
+from ..descriptions import ComposableNode
+
+
+class ComposableNodeContainer(Node):
+    """Action that executes a container ROS node for composable ROS nodes."""
+
+    def __init__(
+        self,
+        *,
+        composable_node_descriptions: Optional[List[ComposableNode]] = None,
+        **kwargs
+    ) -> None:
+        """
+        Construct a ComposableNodeContainer action.
+
+        Most arguments are forwarded to :class:`launch_ros.actions.Node`, so see
+        the documentation of that class for further details.
+
+        :param composable_node_descriptions: optional descriptions of composable nodes to be loaded
+        """
+        super(ComposableNodeContainer, self).__init__(**kwargs)
```

Comment:
nit, `super().__init__(**kwargs)` in python 3

Reply:
That's true, python2 reflexes. Fixed.